examples/multi_label/classifier_chain.py

```diff
@@ -0,0 +1,39 @@
+import numpy as np
```

Comment:
can you create an example that has a file name starting with ``plot_`` and plots things? Plots make for more engaging examples, and only examples that start with ``plot_`` are automatically run. Also, if you want to add a new folder to the examples, you need to create an empty README.txt file in that folder.